installer/pkg/components/jaeger-operator/helm.go

```diff
@@ -18,9 +18,8 @@ var Helm = common.CompositeHelmFunc(
 			Enabled: pointer.BoolDeref(cfg.Config.Jaeger.InCluster, false),
 			Values: &values.Options{
 				Values: []string{
-					"installCRDs=true",
-					"crd.install=false",
-					"rbac.clusterRole=true",
+					helm.KeyValue("jaeger-operator.installCRDs", "true"),
```

Comment:
I _think_ `installCRDs` should be `crd.install` per the operator's [config table](https://github.com/jaegertracing/helm-charts/tree/main/charts/jaeger-operator#configuration). Unless of course I've missing something? Lemme know whatcha think.

I am guessing `installCRDs` is related to prior `cert-manager` [stuffs](https://github.com/gitpod-io/gitpod/blob/main/chart/values.yaml#L758).

Reply:
Yeah, I was rushing at the end of the day. Thanks for saving me from myself